### PR TITLE
Popover timeout handlers

### PR DESCRIPTION
**Pull request by nathansmyth**
Addresses #13.

### CODE REVIEW

File: src/components/popover/PopoverDirective.js [hunk 1]
```diff
@@ -65,13 +65,16 @@ export default function PopoverDirective($q, $timeout, $templateCache,
         showCaret: toBoolean($attributes.popoverPlain || false),
       };
 
-      // attach exit and enter events to element
-      $element.off('mouseenter', display);
+      // attach mouse events to element
       $element.on('mouseenter', display);
-      $element.off('mouseleave', mouseOut);
```
Comment (jonshaffer) [inline src/components/popover/PopoverDirective.js]: add these to $scope.$on('$destroy' ?

Comment (nathansmyth) [inline src/components/popover/PopoverDirective.js]: +1

**Comment by nathansmyth** [comment]
Now addresses #57. Still has an issue in this area.

Also cleaned up some issues. Planning to resolve more tomorrow.

**Comment by jonshaffer** [comment]
More unit tests, any e2e tests would be awesome here

**Comment by nathansmyth** [comment]
+1 for tests

I got distracted improving the code cov score.

**Comment by nathansmyth** [comment]
Got stuck on testing mouseout closing popovers in protractor. If you have any hints, let me know?

Moving to unit testing for now.

**Comment by jonshaffer** [comment]
More info on doing that here:

http://stackoverflow.com/a/28301469/6744582

**Comment by nathansmyth** [comment]
I think I'm done here. I'd like to get to 💯 but all checks are passing and I'm ready to move on and resolve these two issues.

**Pull request merged**